policyengine_us/parameters/gov/contrib/congress/tlaib/income_security_package/end_child_poverty_act/filer_credit/eligibility/max_age.yaml

```diff
@@ -0,0 +1,9 @@
+description: Maximum age for End Child Poverty Act filer credit eligibility
+metadata:
+  label: Maximum age for End Child Poverty Act filer credit eligibility
+  unit: year
+  reference:
+    - title: placeholder - bill not yet introduced
+      href: placeholder
+values:
+  2025-01-01: 65
```

Comment:
2026?

Reply:
I thought that we had some issues with parameter values in structural reforms being set past 2024 (default year) can we for now just use `0000-01-01:` just to be safe